packages/astro/src/@types/astro.ts

```diff
@@ -848,18 +848,32 @@ export interface MarkdownInstance<T extends Record<string, any>> {
 	/** raw Markdown file content, excluding frontmatter */
 	rawContent(): string;
 	/** Markdown file compiled to valid Astro syntax. Queryable with most HTML parsing libraries */
-	compiledContent(): Promise<string>;
-	getHeadings(): Promise<MarkdownHeading[]>;
+	compiledContent(): string;
+	getHeadings(): MarkdownHeading[];
```

Comment:
This sticks out to me. I know the implementation changed to be `sync`, but moving from `sync` to `async` is technically a breaking change.

Did the underlying change happen pre-1.0 and we're just fixing the types?

Reply:
Yep, change happened pre-1.0! We can leave the types as-is if we want, but it shouldn't effect any end user behavior.